setup.py

```diff
@@ -148,6 +148,10 @@ def run(self):
         'ujson>=4.0.1',
         'aiofiles>=0.6.0',
         'starlette==0.14.0',
+        'zarr>=2.5.0',
+        'numcodecs>=0.5.7',
+        'scipy>=1.2.1',
+        'negspy>=0.2.24',
```

Comment:
Should this be added to the `environment.yml`?

Reply:
(Both in `notebooks` and root)

Reply:
Not necessarily, these should be automatically installed when the `vitessce` package is installed since they are listed here

Reply:
I kept getting an error about `negspy` not being found.  I am going to try a clean clone of the repo.